### PR TITLE
Add myself in .git-authors

### PR DESCRIPTION
**Pull request by markphd**
Initials, name, github username, and email address added.

### CODE REVIEW

File: .git-authors [hunk 1]
```diff
@@ -8,6 +8,7 @@ pairs:
   nw: Nathan Wang; nathan.wang2
   sc: Serena Cao; wcao0630
   mw: Matt Wills; mattwills8
+  md: Mark Philip Dawal; markphd
```
Comment (bbmoz) [inline .git-authors]: I think you and Matt don't have to put in the part after the `;`. Could you fix this for yourself and Matt? Make sure you can `git solo`.

Comment (markphd) [reply .git-authors]: Should be good now. I tried `git solo` and it shows:
```
GIT_AUTHOR_NAME='Mark Philip Dawal'
GIT_AUTHOR_EMAIL='mark@forloop.me'
```
So I think everything is okay now? 😄

File: .git-authors [hunk 2]
```diff
@@ -7,7 +7,8 @@ pairs:
   km: Klaus Mana
   nw: Nathan Wang; nathan.wang2
   sc: Serena Cao; wcao0630
-  mw: Matt Wills; mattwills8
+  mw: Matt Wills;
+  md: Mark Philip Dawal;
```
Comment (fybwid) [inline .git-authors]: Add your Gmail address here ~`@gmail`~

File: .git-authors [hunk 3]
```diff
@@ -18,3 +19,4 @@ email_addresses:
    kj: karthijey@yahoo.com
    km: klausmana@outlook.com
    mw: matt_wills8@hotmail.co.uk
+   md: radical.mark@gmail.com
```
Comment (fybwid) [inline .git-authors]: Remove this line because the default email setting used `@gmail.com`

File: .git-authors (hunk 2 above)
Comment (fybwid) [reply .git-authors]: @bbmoz Mark Philip Dawal used `@gmail.com` so @markphd only need to add the Gmail address.
For @mattwills8 need to remove `;` after `mw: Matt Wills`

Comment (markphd) [reply .git-authors]: Ok, should be updated now

**Comment by bbmoz** [comment]
Hi Mark, too many files are changes here. Please re-create a pull request with just the 1-file change. I suspect that a merge overwrote the git history.

**Pull request closed without merging**